### PR TITLE
Add localization to plugins under qa-plugin

### PR DESCRIPTION
**Pull request by jairlopez**
#706

### CODE REVIEW

File: qa-plugin/wysiwyg-editor/qa-plugin.php [hunk 1]
```diff
@@ -42,5 +42,6 @@
 
 qa_register_plugin_module('editor', 'qa-wysiwyg-editor.php', 'qa_wysiwyg_editor', 'WYSIWYG Editor');
 qa_register_plugin_module('page', 'qa-wysiwyg-upload.php', 'qa_wysiwyg_upload', 'WYSIWYG Upload');
+qa_register_plugin_phrases('qa-wysiwyg-lang-*.php', 'wysiwyg');
```
Comment (pupi1985) [inline qa-plugin/wysiwyg-editor/qa-plugin.php]: Should the `'page'` modules be together followed by the `_phrases` or is my TOC kicking again? :)

**Comment by pupi1985** [comment]
Looks much better to me :)

BTW, I've just realized that you don't add commas after the last element of an array. I've configured my IDE to add them automatically. I think Scott adds them too but maybe he can confirm that so that we avoid overwriting commas when formatting each other's code.

**Comment by svivian** [comment]
Wow, kudos for the swift response! All looks good from a cursory glance on mobile, will merge in the next few days.

Re: arrays, yeah I usually do add trailing commas on multi-line arrays. I guess we don’t have a codesniffer rule for that implemented. Not a major issue though, I only spotted 2 instances anyway.

**Comment by svivian** [comment]
Thanks again for this, good work!

Just an FYI, the language files should not have HTML in them. The way to use HTML is to pass that in with the substitution. I've made an update in da09b279e4d1e323f277b951607f9407c469fdf0 that does this.

**Comment by jairlopez** [comment]
Awesome! Thank you for that.

**Pull request merged**